rules/manager.go

```diff
@@ -35,9 +35,11 @@ import (
 	"github.com/prometheus/prometheus/model/rulefmt"
 	"github.com/prometheus/prometheus/model/timestamp"
 	"github.com/prometheus/prometheus/model/value"
+	"github.com/prometheus/prometheus/notifier"
```

Comment:
I wonder if having `sendAlerts` in the main package was intended to separate `manager` from depending on `notifier` ?

Reply:
Possible. We can revisit this again. It can be moved to a util package.